java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -247,4 +252,47 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         String data = client.set("another", ANOTHER_VALUE, options).get();
         assertNull(data);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void sadd_srem_scard_smembers_existing_set(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        assertEquals(4, client.sadd(key, "member1", "member2", "member3", "member4"));
```

Comment:
Add .get() to actually complete the futures you're IT tests are failing because of that.